site/docs/v1/tech/reactor.adoc

```diff
@@ -24,11 +24,20 @@ Once a license is purchased, in order to activate your instance and enable React
 
 Then in your FusionAuth instance navigate to [breadcrumb]#Reactor# and enter your License Id under *Start the Reactor*.
 
-_Note: You may enter your License Id on multiple instances.
-However, since billing is based upon the total number of monthly active users across all licensed instances it is recommended that you only license those instances you need support for or require Reactor features for._
+[NOTE]
+====
+In an link:https://en.wikipedia.org/wiki/Air_gap_(networking)[air gapped] configuration where outbound network access is not available, the license text will be available in your link:https://account.fusionauth.io/account[Account] as well. Include this in addition to the License Id when activating.
+====
```

Comment:
I don't love the two notes together, but if there's no verbiage that naturally fits there, I guess it is ok.

Reply:
"Include this in addition to the License Id when activating." 

Is this true? do you need to provide both?

Reply:
Well the API does not null validation on the licenseId so its true as far as making a successful request. Beyond that I'm not sure.

Reply:
Okay, fair enough. Sounds like you need to supply it.

Reply:
LGTM